tests/test_tip_endpoint.py

```diff
@@ -0,0 +1,10 @@
+from tests.helpers.requests import get_request
```

Comment:
tip endpoint could easily fail data validations, as two instances could have their tip incremented while the test is running, if we're validating script > tx > block endpoint, we're already validating that the randomly elected block is present on target environment, and thus - could be avoided (besides, tip is being validated by grest-poll)

Reply:
Yep, we mentioned this before. grest-poll does validate tip but not to this extent. And I agree that this is a test that's easy to fail because of the nature blockchain syncing, on guild I have not seen it fail even once I think, but mainnet is most likely a different story.

I'll remove it 👍